src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/OnboardingSupervisor.kt

```diff
@@ -884,6 +884,176 @@ internal class OnboardingSupervisor(
         }
     }
 
+    @Suppress("UnusedPrivateMember")
+    private fun retrieveSkipWithdrawalRouteExchange(
+        state: PerpetualState?,
+        decimals: Int,
+        gas: BigDecimal,
+        accountAddress: String,
+        sourceAddress: String,
+        subaccountNumber: Int?,
+    ) {
+        val usdcSize = helper.parser.asDecimal(state?.input?.transfer?.size?.usdcSize)
+        val fromAmount = if (usdcSize != null && usdcSize > gas) {
+            ((usdcSize - gas) * Numeric.decimal.TEN.pow(decimals)).toBigInteger()
+        } else {
+            null
+        }
+        val chainId = helper.environment.dydxChainId
+        val squidIntegratorId = helper.environment.squidIntegratorId
+        val dydxTokenDemon = helper.environment.tokens["usdc"]?.denom
```

Comment:
Can we use nativeTokenDemon instead?  Legal prefers "native" over "dydx" in our codebase.

Also, tokens["usdc"] would be the UDSC instead Native token, so the variable name should change accordingly.

Reply:
ah got it thanks! this was c/p'ed from another method and was confused by this naming as well. thanks :)

Reply:
renamed to `nativeChainUSDCDenom` since `nativeToken` isn't quite accurate.